db/chain/chain_test.go

```diff
@@ -98,10 +98,10 @@ func TestExpresionChain_Render(t *testing.T) {
 		},
 		{
 			name: "basic selection with distinct",
-			chain: (&ExpresionChain{}).Select(Distinct("field1")).
+			chain: (&ExpresionChain{}).Select(As(Distinct("field1"), "renamed")).
```

Comment:
ok, this is sort of wrong, this should work without AS so I would Add a test instead of changing this one (and by add I mean blatantly copy this one and change it)

Reply:
It does work without, I'll make a second copy. Just wanted to check that two modifiers work together too.